app/auth/login-redirect/route.ts

```diff
@@ -17,14 +17,17 @@ export async function GET(request: NextRequest) {
   const requestUrl = getRequestUrl(request);
   const { searchParams } = new URL(requestUrl);
   const originalUrl = searchParams.get(AUTH_LOGIN_CALLBACK_PARAM);
+  const prompt = (searchParams.get('prompt') as 'login' | 'none') ?? 'login';
```

Comment:
just create a const so that people would understand that the 'none' here corresponds to the one in the middleware